Client/Frontend/Browser/Playlist/Managers & Cache/PlaylistCarplayManager.swift

```diff
@@ -64,107 +51,45 @@ class PlaylistCarplayManager: NSObject {
     // in use at any given moment
     static let shared = PlaylistCarplayManager()
     
-    private override init() {
-        super.init()
-        
-        if #available(iOS 14, *) {
-            // iOS 14 does NOT require initialization :)
-            // This is because the CPTemplate gets called properly
-            // So we know exactly when CarPlay is connected, and when it is disconnected.
-            return
-        } else {
-            contentManager = MPPlayableContentManager.shared()
-        }
-        
-        #if targetEnvironment(simulator)
-        // Force CarPlay on the simulator
-        // This is because we don't actually know if the CarPlay simulator is showing or not
-        // and we don't get notified when it disconnects :(
-        attemptInterfaceConnection(isCarPlayAvailable: true)
-        #else
-        guard let contentManager = contentManager else {
-            log.error("Invalid MPPlayableContentManager!")
-            return
+    func getCarPlayController() -> NSObject? {
```

Comment:
Can we return the real value now instead of `NSObject` that its iOS 14+ and only has 1 implementation?